templates/module/src/entity-route-provider.php.twig

```diff
@@ -29,39 +29,8 @@ class {{ entity_class }}HtmlRouteProvider extends AdminHtmlRouteProvider {% endb
   public function getRoutes(EntityTypeInterface $entity_type) {
     $collection = parent::getRoutes($entity_type);
 
-    $entity_type_id = $entity_type->id();
-
-    if ($collection_route = $this->getCollectionRoute($entity_type)) {
-      $collection->add("entity.{$entity_type_id}.collection", $collection_route);
-    }
+    // Provide your custom entity routes here.
```

Comment:
For config entities the `HtmlRouteProvider` has now become unnecessary. I have replaced the logic with this comment that explains that custom routes can be added here.

Shall we keep this dev hint or would it be better to just remove the file?